mautic/mautic
# Revert back SlimScroll library to Version: 1.3.2 - fix scroll issue

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements / 3.0 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 




#### Description:
I find the library what make ugly middle button scroll issue.
I've noticed it made this @anton-vlasenko  update of slimScroll library  https://github.com/mautic/mautic/commit/e7fb2616bf9bac092bb2b16e0a3503fbdcd3da00#diff-1175126cdcb0bf74d4535df8bd48e5940235520ddff2be4a4e4c9cd4177a2355 
It's not clear what Anton did make a changes, but I made a research and this comment fix issue from original 1.3.2 version - https://github.com/rochal/jQuery-slimScroll/issues/316#issuecomment-566278726  



#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Change your window size to see slim scroll on left navigation menu 
3. Then try scroll in the navigation menu and try scroll in all page
4. Scrolling doesn't work
5. After patch should work

![image](https://user-images.githubusercontent.com/462477/96332815-83ddec80-1066-11eb-8cf1-d33d8c654f50.png)
